bin/ch/ChakraRtInterface.cpp

```diff
@@ -138,13 +133,15 @@ bool ChakraRTInterface::LoadChakraDll(ArgInfo* argInfo, HINSTANCE *outLibrary)
     m_jsApiHooks.pfJsrtDiagGetObjectFromHandle = (JsAPIHooks::JsrtDiagGetObjectFromHandle)GetChakraCoreSymbol(library, "JsDiagGetObjectFromHandle");
 
 #ifdef _WIN32
```

Comment:
Yes. It's not yet xplat ready. New method is just for keeping wchar_t from ch entirely.